runtime-sdk/src/runtime.rs

```diff
@@ -19,6 +19,9 @@ pub trait Runtime {
     /// State version.
     const STATE_VERSION: u32 = 0;
 
+    /// Prefetch limit. To disable prefetch set it to 0.
+    const PREFETCH_LIMIT: u16 = std::u16::MAX;
```

Comment:
I remember reading in one of your comments that enabling prefetch worsens benchmark results in some cases? Does it still make sense to enable it by default?

Reply:
By about 5% on the accounts-transfer benchmark yes. Agreed it makes more sense to disable it by default for now.